src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -584,7 +584,7 @@ const DataAccessRequestApplication = (props) => {
                     <ProgressReportApplication
                       readOnlyMode={false}
                       datasets={datasets}
-                      dar={reverseOrderedDARs[0]}
+                      dar={merge(reverseOrderedDARs[0]?.data, reverseOrderedDARs[0])}
```

Comment:
I like the preservation of the outer object's (second parameter) values as truth!  I think that's the right call.